.travis.yml

```
@@ -54,6 +54,22 @@ matrix:
             - make
             - cd ..
           env: BUILD_TYPE=Release
+          
+         # Power jobs
+        - os: linux
+          arch: ppc64le
+          env:
+          - BUILD_TYPE=Debug
+          - BUILD_OPTS='-DENABLE_CODE_COVERAGE=ON -DENABLE_EXPERIMENTAL_BONDING=ON'
+          - RUN_SONARCUBE=1
+          - RUN_CODECOV=1
+        - arch: ppc64le
+          env:
+          - BUILD_TYPE=Debug
+          - BUILD_OPTS='-DENABLE_LOGGING=OFF -DENABLE_MONOTONIC_CLOCK=ON -DENABLE_EXPERIMENTAL_BONDING=ON'
+        - os: linux
+          arch: ppc64le
+          env: BUILD_TYPE=Release
```

The must be only one job that runs SonarCube and CodeCov, as the results are uploaded to those systems.
What about having just two build configurations for ppc64e: debug and release?

```suggestion
         # Power jobs
        - os: linux
          arch: ppc64le
          env:
          - BUILD_TYPE=Debug
        - arch: ppc64le
          env:
          - BUILD_TYPE=Release
          - BUILD_OPTS='-DENABLE_MONOTONIC_CLOCK=ON'
```